data/mods/Magiclysm/Spells/attunements/Alchemist.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "id": "alchemist_make_potion_bomb",
+    "type": "SPELL",
+    "name": "Manafusion bomb",
+    "description": "Oversaturates potion starter with mana to create a deadly mana explosion.",
+    "valid_targets": [ "none" ],
+    "flags": [ "SOMATIC", "WONDER" ],
+    "effect": "recover_energy",
+    "effect_str": "STAMINA",
+    "//": "I almost forgot how i make this spell, so: you cast spell, main one give you a little of stamina, and call secondary spell that create bomb. I make it like that to avoid bug with activated items and timer mechanic incompability and fact that PERMANENT flag works only on max lvl spell.",
+    "components": "spell_components_manabomb",
+    "shape": "blast",
+    "min_damage": 1,
+    "max_damage": 3,
+    "damage_increment": 0.07,
+    "max_level": 35,
+    "spell_class": "ALCHEMIST",
+    "energy_source": "MANA",
+    "base_casting_time": 180000,
+    "final_casting_time": 12000,
+    "casting_time_increment": -5000,
+    "base_energy_cost": 400,
+    "final_energy_cost": 800,
+    "energy_increment": 12,
+    "extra_effects": [ { "id": "alchemist_make_potion_bomb_create" } ],
+    "difficulty": 8
+  },
+  {
+    "id": "alchemist_make_potion_bomb_create",
+    "type": "SPELL",
+    "name": "Manafusion bomb create",
+    "description": "Gives you a manafusion bomb.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "none" ],
+    "flags": [ "PERMANENT" ],
+    "effect": "spawn_item",
+    "effect_str": "alchemist_bomb",
+    "shape": "blast",
+    "min_damage": 1,
+    "max_damage": 1
+  },
+  {
+    "id": "alchemist_make_potion_trap",
+    "type": "SPELL",
+    "name": "Manafusion trap",
+    "description": "Make a manafusion trap, stabilizing manafusion bombs and giving you some of the excess mana.",
+    "//": "In perfect world there will be another spell that create already buried mine, making very effective trap in realtime. But not today.",
+    "valid_targets": [ "none" ],
+    "flags": [ "SOMATIC", "WONDER" ],
+    "effect": "recover_energy",
+    "effect_str": "STAMINA",
+    "shape": "blast",
+    "components": "spell_components_manatrap",
+    "min_damage": 1,
+    "max_damage": 3,
+    "damage_increment": 0.07,
```

Comment:
same as above. if you really wanted this to be a placeholder it should have been an attack spell that does 0 damage...